clock.js

```diff
@@ -161,4 +165,27 @@ browser.runtime.onMessage.addListener( (request, sender, sendResponse) => {
 			return Promise.resolve( { response: "shown" } );
 		}
 	}
+	if( request.recreate == "true" ) {
+		chrome.storage.sync.get(document.clock_defaults, create_clock);
+	}
 } );
+
+function check_is_page_in_domains_list( domains ) {
```

Comment:
The domain detection thing is very confusing, because Chrome does not display the full name. E.g. when the URL box shows `ibm.com/uk-en` the real URL is `https://www.ibm.com/uk-en` (with a `www`). I spent some time not understanding the behavior before I realized that.

So instead of having the user enter domain names, maybe add a button "Exclude this domain"?

Reply:
Yeah, chrome thingies, who would know, that they will be obstacles ¯\_(ツ)_/¯

> maybe add a button "Exclude this domain"?

Yes, would be conveniently, if somebody would add it.